lib/improver/tests/utilities/test_spatial.py

```diff
@@ -52,8 +52,7 @@
     check_if_grid_is_equal_area, calculate_grid_spacing,
     convert_distance_into_number_of_grid_cells,
     convert_number_of_grid_cells_into_distance,
-    lat_lon_determine, lat_lon_transform, transform_grid_to_lat_lon,
-    get_nearest_coords)
+    lat_lon_determine, transform_grid_to_lat_lon)
```

Comment:
It looks like the `get_nearest_coords` function hasn't been removed, only the unit tests. Could the `get_nearest_coords` function be removed?

Reply:
My bad, made a change and didn't check at the end. will push changes up now